docs/pages/accounts/account-types.md

```diff
@@ -53,19 +69,19 @@ We have taken a lot of care to make sure that all of the functionality that you
 
 ### Renaming an Account
 
-Accounts may be renamed a limited number of times. Simply visit [the account settings page](https://expo.dev/accounts/[account]/settings) and follow the prompts under **Rename Account**.
+Accounts can be renamed a limited number of times. Visit [the account settings](https://expo.dev/accounts/[account]/settings) and follow the steps under [Rename Account](https://expo.dev/settings#rename-account).
 
-#### Some caveats
+<ImageSpotlight alt="Rename account settings." src="/static/images/accounts/rename-account.jpg" style={{maxWidth: 720}} />
 
-- New publishes for projects belonging to renamed accounts must be on SDK 43 or higher.
+One caveat to remember when renaming accounts is that the new app published for your projects belonging to renamed accounts must be on SDK 43 or higher.
 
-### Transferring Projects Between Accounts
+## Transferring Projects Between Accounts
 
-Projects may be renamed a limited number of times. Simply visit [the project settings page](https://expo.dev/accounts/[account]/projects/[project]/settings) and follow the prompts under **Transfer project**.
+Projects can be transferred a limited number of times. Visit [the project settings page](https://expo.dev/accounts/[account]/projects/[project]/settings) and follow the steps under **Transfer project**.
 
 #### Some caveats
 
-- The person performing the transfer must have "Owner" role on both the source and destination accounts.
-- New publishes for renamed project must be on SDK 43 or higher.
+- The person performing the transfer must have the Owner role on both the source and destination accounts.
+- New publish for a renamed project must be on SDK 43 or higher.
 
-> 💡 In the case where a project on your Personal or Organization Account (`source`) is sold/given to another company/person (receiving party) and the receiving party does not want to grant you Owner access to their `destination` account, you can create a new Organization Account (`escrow`), grant the receiving party Owner access, and transfer the project to the `escrow` account. The receiving party then can transfer it to the `destination` account from the `escrow` account.
+> In a scenario where a project on your Personal or Organization Account (`source`) is sold/given to another company/person (receiving party), and the receiving party does not want to grant you Owner access to their `destination` account, you can create a new Organization Account (`escrow`), grant the receiving party Owner access, and transfer the project to the `escrow` account. The receiving party then can transfer it to the `destination` account from the `escrow` account.
```

Comment:
I would simplify this a lot: 'If you want to transfer ownership of a project, but you can't be given Admin permissions on the destination account, you can create an escrow account etc. etc.'